test-suite/src/function/entries.rs

```diff
@@ -0,0 +1,28 @@
+use {
+    crate::*,
+    gluesql_core::prelude::{Payload, Value::*},
+};
+
+test_case!(entries, async move {
+    test! {
+        name: "test entries function works while creating a table simultaneously",
+        sql: "CREATE TABLE Item (map MAP)",
+        expected: Ok(Payload::Create)
+    };
+    test! {
+        name: "test if the sample string gets inserted to table",
+        sql: "INSERT INTO Item VALUES ('{\"name\":\"GlueSQL\"}')",
```

Comment:
we can use r# operator to remove '\' for readable code
```rust
r#"INSERT INTO Item VALUES ('{"name":"GlueSQL"}')"#,
```

Reply:
thanks for the feedback. i've applied it :)